wgsl/index.bs

```diff
@@ -489,8 +489,11 @@ The following is a list of keywords which are reserved for future expansion.
   <tr><td>`NOT_EQUAL`<td>!==
   <tr><td>`GREATER_THAN`<td>>
   <tr><td>`GREATER_THAN_EQUAL`<td>>=
+  <tr><td>`LOGICAL_SHIFT_RIGHT`<td>>>
+  <tr><td>`ARITH_SHIFT_RIGHT`<td>>>>
   <tr><td>`LESS_THAN`<td><
```

Comment:
Might want to consider call this `ANGLE_BRACKET_LEFT` and `ANGLE_BRACKET_RIGHT` to separate terminology between lexing and parsing.

That would be more consistent with `LEFT_BRACE` instead of `BEGIN_SCOPE`, `BANG` instead of `NOT`, `LEFT_BRACKET` instead of `BEING_ARRAY_INDEX`.